Content.Client/Chat/UI/SpeechBubble.cs

```diff
@@ -118,12 +125,6 @@ protected override void FrameUpdate(FrameEventArgs args)
                 Modulate = Color.White;
             }
 
-            if (_entityManager.Deleted(_senderEntity) || _timeLeft <= 0)
-            {
-                // Timer spawn to prevent concurrent modification exception.
-                Timer.Spawn(0, Die);
-                return;
-            }
 
             var worldPos = _entityManager.GetComponent<TransformComponent>(_senderEntity).WorldPosition;
```

Comment:
can't you use the cached `TransformComponent` here?

Reply:
to be entirely honest: I didn't even look that far down